### PR TITLE
PR for public windows IoT resource provider

### PR DESCRIPTION
**Pull request by bfjelds**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by bfjelds** [comment]
Link to PR/review in azure-rest-api-specs-pr: https://github.com/Azure/azure-rest-api-specs-pr/pull/434

**Comment by dsgouda** [comment]
@sergey-shandar I'll handle this

File: specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false,
```
Comment (dsgouda) [inline specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json]: Please check if this is a long running operation

File: specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
```
Comment (dsgouda) [inline specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json]: Please check if this is a long running operation

File: specification/windowsiot/resource-manager/readme.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+These settings apply only when `--tag=package-2018-02` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-02'
+input-file:
+- Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json
+```
+
+---
+# Code Generation
```
Comment (dsgouda) [inline specification/windowsiot/resource-manager/readme.md]: If you intend to generate C#/other SDKs please add specific settings here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great for the most part. I understand this has been reviewed before but need to check a few things

File: specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json (hunk 1 above)
Comment (bfjelds) [reply specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json]: it is not a long running operation.

File: specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json (hunk 2 above)
Comment (bfjelds) [reply specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json]: it is not a long running operation.

File: specification/windowsiot/resource-manager/readme.md (hunk 3 above)
Comment (bfjelds) [reply specification/windowsiot/resource-manager/readme.md]: no SDKs being generated.

**Review by dsgouda — APPROVED** [review]
Thanks for the feedback

**Pull request merged**